x-pack/plugins/uptime/server/lib/requests/get_certs.ts

```diff
@@ -67,6 +67,11 @@ export const getCerts: UMElasticsearchQueryFn<GetCertsParams, CertResult> = asyn
               field: 'tls.server',
             },
           },
+          {
+            exists: {
+              field: 'tls.server.hash.sha256',
```

Comment:
why not modify the existing `exists` filter on line 67?

Reply:
I suppose there wouldn't be any downside - the query won't work unless `sha256` is present. I guess in the moment I was thinking it covers the remaining `tls.server.*` fields we rely on, but checking for `sha256` is probably what we should have done in the first place.

Reply:
c03115b6abe0574a38598bdfdf68dfec067f56b8

Reply:
instead of modifying you deleted it :O